README.md

```diff
@@ -40,7 +40,8 @@ which :: Dynamic Bool
 (which >>= if _ then d1 else d2) :: Dynamic Int
 ```
 
-We can introduce new Dynamics using `newDynamic`:
+We can introduce new Dynamics using `newDynamic`, this is only required for top level components/widgets. Root dynamics
```

Comment:
> this is only required for top level components/widgets

What do you mean by this? "components" are only a convention, it's just function calls

Reply:
You are of course right, probably has to be renamed to root dynamics, maybe with an explanation.

Reply:
Done, please have a look.